main.nf

```diff
@@ -2257,14 +2256,19 @@ process annotsv {
 	output:
 		set group, file("${group}_annotsv.tsv") into annotsv, annotsv_ma, annotsv_fa
 
-	"""
-	export ANNOTSV="/AnnotSV"
-	/AnnotSV/bin/AnnotSV -SvinputFile $sv \\
-		-typeOfAnnotation full \\
-		-outputDir $group \\
-		-genomeBuild GRCh38
-	mv $group/*.annotated.tsv ${group}_annotsv.tsv
-	"""
+	shell:
+		'''
+		export ANNOTSV="/AnnotSV"
+		/AnnotSV/bin/AnnotSV -SvinputFile !{sv} \\
+			-typeOfAnnotation full \\
+			-outputDir !{group} \\
+			-genomeBuild GRCh38
+		if [-f !{group}/*.annotated.tsv]; then
+			mv !{group}/*.annotated.tsv !{group}_annotsv.tsv
```

Comment:
Is the idea here to rename a collection of files? I.e.

```
group1/aaa.annotated.tsv -> group1_annotsv.tsv
group2/aaa.annotated.tsv -> group2_annotsv.tsv
```

This would not work in bash (except for if you end up with only one file), but I am not familiar with how nextflow uses the `!{group}` syntax. So if you are more familiar with it and have tested, then it is probably no problem.

Reply:
this is a stupid thing I do to create an output file when there is no output from AnnotSV. When the vcf is empty AnnotSV just exits without creating a *.annotated.tsv. But just touch the file makes prescore.pl bug out and not add headers to the vcf, which in turn makes genmod score bug out cause the vcf is not valid.

Reply:
OK I think I get it, I was thinking you were squeezing multiple groups into the `!{group}`, but I see now that there will be only one per process. I understand the logic of the else branch.

Could be a nice place to leave an explanative comment :)